python/src/scipp/table_html/formatting_html.py

```diff
@@ -173,15 +173,41 @@ def _icon(icon_name):
             "</svg>".format(icon_name))
 
 
-def summarize_coord(dim, var):
+def summarize_coord(dim, var, ds=None):
     is_index = dim in var.dims
-    return summarize_variable(str(dim), var, is_index)
+    bin_edges = find_bin_edges(var, ds) if ds else None
+    return summarize_variable(str(dim), var, is_index, bin_edges=bin_edges)
 
 
-def summarize_coords(coords):
+def find_bin_edges(var, ds):
+    """
+    Checks if the coordinate contains bin-edges.
+    """
+    bin_edges = []
+    for idx, dim in enumerate(var.dims):
+        len = var.shape[idx]
+        if dim in ds.dims:
+            if ds.shape[ds.dims.index(dim)] + 1 == len:
+                bin_edges.append(dim)
+    return bin_edges
+
+
+def summarize_coords(coords, ds=None):
+    vars_li = "".join(
+        "<li class='xr-var-item'>"
+        f"{summarize_coord(dim, var, ds)}"
+        "</span></li>"
+        for dim, var in coords)
+
+    return f"<ul class='xr-var-list'>{vars_li}</ul>"
+
+
+def summarize_labels_masks(data):
```

Comment:
Why the special treatment for coords? There are in principle no restrictions about what can be on the bin edges, i.e, you should handle labels and masks in the same way (and also data).

Reply:
👍 I removed that func, now they all go through `summarize_coords`